src/js/containers/state/StateContainer.jsx

```diff
@@ -89,7 +89,7 @@ const StateContainer = (props) => {
         const [wasInputStateName, stateName, stateId] = parseStateDataFromUrl(state);
 
         if (!Object.keys(props.match.params).includes('fy')) {
-            // props.history.replace(`/state/${stateName}/latest`);
```

Comment:
We should keep the above commented out code.  Line #93 was changed due to a temporary backend change.  When the backend issue is resolved, we should go back to the commented out code.

Reply:
i added the commented out code back in